app/src/renderer/components/network/PageNetwork.vue

```diff
@@ -132,7 +132,10 @@ export default {
         }
       ]
     }
-  }
+  },
+  async mounted() {
```

Comment:
nice!

this will only get the data once... right? should we update it every block / every 10 blocks? is this too many requests?

Reply:
issue states: `* rewards should update in the UI as they update`